dhcpv4/modifiers_test.go

```diff
@@ -39,9 +39,19 @@ func TestWithOptionModifier(t *testing.T) {
 	d, err := New(WithOption(OptDomainName("slackware.it")))
 	require.NoError(t, err)
 
-	dnOpt := d.DomainName()
```

Comment:
Actually, this change is not related to the title of the PR. Do you mind having these changes separately?

Reply:
I can remove them from this PR.

But at this point i would like to see a `CONTRIBUTING.md` or similar guideline explaining for example that PRs must _only_ contain minimal changes that are directly related to the description (and any other project policies). To be clear, I've got nothing against this policy (or any other for that matter), but the contribution workflow has to be clear to contributors _before_ they start their work to avoid wasting everyone's time with these tiny incremental reviews. (And no, it is not obvious as each project does things a bit differently.)